docker-compose.yaml

```diff
@@ -6,21 +6,21 @@ services:
     volumes:
       - postgres_data:/var/lib/postgresql/data
     environment:
-      - POSTGRES_USER=smallboard_user
-      - POSTGRES_PASSWORD=smallboard_password
-      - POSTGRES_DB=smallboard
+      - POSTGRES_USER=cardboard_user
+      - POSTGRES_PASSWORD=cardboard_password
+      - POSTGRES_DB=cardboard
   redis:
     image: redis:alpine
   web:
     build: .
     command: python manage.py runserver 0.0.0.0:8000
     volumes:
-      - .:/usr/src/smallboard/:Z
-      - /usr/src/smallboard/node_modules
+      - .:/usr/src/cardboard/:Z
+      - /usr/src/cardboard/node_modules
     ports:
       - 8000:8000
     environment:
-      - DATABASE_URL=postgres://smallboard_user:smallboard_password@db/smallboard
+      - DATABASE_URL=postgres://cardboard_user:cardboard_password@db/cardboard
```

Comment:
This is a breaking change locally. Doesn't affect me personally though since I just moved to a new laptop and nuked my old Docker data, though.

Reply:
Ah okay, should I keep docker-compose.yaml the same?

Reply:
Changing it is probably fine (doubt anyone else both has local data they care about + is using docker), just worth keeping in mind.